### PR TITLE
Changelog for v2.122.0

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,24 +4,104 @@
...
+
+  ``risk:compromise``
+    Add a ``severity`` property that can be used as a relative severity score
+    for the compromise.
+
+  ``risk:threat``
+    Add a ``type`` property to trecord the type of the threat cluster.
+
+  ``risk:threat:type:taxonomy``
+    Add a form to record an taxonomy of threat types.
```
Comment (invisig0th) [inline CHANGELOG.rst:68]: a taxonomy

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,24 +4,104 @@
...
 - Add support for Python 3.10 to Synapse.
   (`#2962 <https://github.com/vertexproject/synapse/pull/2962>`_)
 - Update the Synapse docker containers to be built from a Debian based image,
   instead of an Ubuntu based image.
   (`#2962 <https://github.com/vertexproject/synapse/pull/2962>`_)
-
-Bugfixes
---------
-TBD
+- Add an optional ``--type`` argument to the Storm ``node.add`` command.
```
Comment (invisig0th) [inline CHANGELOG.rst:75]: note.add

**Review by invisig0th — DISMISSED** [review]

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,24 +4,104 @@
...
+
+  ``file:bytes``
+    Add ``exe:compiler`` and ``exe:packer`` properties to track the software
+    used to compile and encode the file.
+
+  ``lang:language``
+    Add a new guid form to represent a written or spoken language.
+
+  ``lang:name``
+    Add a new form record the name of a language.
```
Comment (Cisphyx) [inline CHANGELOG.rst:29]: "to" record

File: CHANGELOG.rst [hunk 4]
```diff
@@ -4,24 +4,104 @@
...
+
+  ``ps:skill``
+    Add a form to record a specific skill which a person or organization may
+    have.
+
+  ``ps:skill:type:taxonomy``
+    Add a form to record a taxonomy of skill types.
+
+  ``ps:proficiency``
+    Add a form to record the assessment that a given contact possess a
```
Comment (Cisphyx) [inline CHANGELOG.rst:54]: possesses

File: CHANGELOG.rst [hunk 5]
```diff
@@ -4,24 +4,104 @@
...
 - Update the Synapse docker containers to be built from a Debian based image,
   instead of an Ubuntu based image.
   (`#2962 <https://github.com/vertexproject/synapse/pull/2962>`_)
-
-Bugfixes
---------
-TBD
+- Add an optional ``--type`` argument to the Storm ``note.add`` command.
+  (`#2984 <https://github.com/vertexproject/synapse/pull/2984>`_)
+- Add a Storm command, ``gen.lang.language``, to lift oer generate a
```
Comment (Cisphyx) [inline CHANGELOG.rst:77]: "oer"

**Review by Cisphyx — DISMISSED** [review]

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**